### PR TITLE
feat: Add support for aws_db_subnet_group resources

### PR DESCRIPTION
**Pull request by mariux**

### CODE REVIEW

**Review by soerenmartius — CHANGES_REQUESTED** [review]
Changelog update is missing

**Comment by soerenmartius** [comment]
The feature should be listed in https://github.com/mineiros-io/terraform-aws-vpc#module-features also

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ that enables our users and customers to easily deploy and manage reusable,
 secure, and production-grade cloud infrastructure.
 
 - [Module Features](#module-features)
-- [Getting Started](#getting-started)
+- [Getting Startci_service_account_nameed](#getting-started)
```
Comment (soerenmartius) [inline README.md:21]: copy & paste issue?

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -61,7 +62,7 @@ Additional features can be enabled on demand.
   Cross Account VPC Peering support,
   IPv6 support,
   VPC Flow Logs,
-  Subnet group support for AWS services like RDS and Redis,
+  Subnet group support for Redis,
```
Comment (soerenmartius) [inline README.md:65]: Did we really only add subnet group support for Redis?

File: README.md [hunk 3]
```diff
@@ -269,6 +270,11 @@ See [variables.tf] and [examples/] for details and use-cases.
   }
   ```
 
+- **`db_subnet_group_name`**: _(Optional `string`)_
+
+  The name of a db subnet group to create for all netnum ranges in this subnet.
+  The db_subnet_group_name resource tags will be cloned from the subnets.
```
Comment (soerenmartius) [inline README.md:276]:
```suggestion
  The `aws_db_subnet_group ` resource tags will be cloned from the subnets.
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mariux) [reply README.md:21]: saw it, excluded it and used `commit -a` as it seems... stupid me... haha

File: README.md (hunk 2 above)
Comment (mariux) [reply README.md:65]: no, we only added it for RDS now... elasticache is missing..as all the others are also still missing..

File: README.md (hunk 3 above)
Comment (mariux) [inline README.md:276]:
```suggestion
  The `db_subnet_group_name` resource tags will be cloned from the subnets.
```

File: README.md (hunk 1 above)
Comment (mariux) [inline README.md:21]:
```suggestion
- [Getting Started](#getting-started)
```

File: README.md (hunk 2 above)
Comment (soerenmartius) [reply README.md:65]: so then it should say RDS, right? 

>   Subnet group support for Redis,

Comment (soerenmartius) [inline README.md:65]:
```suggestion
  Subnet group support for AWS RDS,
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mariux) [reply README.md:65]: that's missing features

Comment (mariux) [reply README.md:65]: dismissed ;)

Comment (soerenmartius) [reply README.md:65]: ah true :)

**Review by soerenmartius — APPROVED** [review]
lgtm!

**Pull request merged**